pkg/api/summary.go

```diff
@@ -56,59 +51,44 @@ func (c *Client) Summaries(startDate, endDate time.Time) ([]summary.Summary, err
 		))
 	}
 
-	summaries, err := ParseSummariesResponse(body)
+	summary, err := ParseSummaryResponse(body)
```

Comment:
We are parsing the result from `/users/current/statusbar/today`.
```suggestion
	today, err := ParseTodayResponse(body)
```